onnxruntime/core/session/environment.cc

```diff
@@ -482,36 +507,25 @@ Status Environment::EpInfo::Create(std::unique_ptr<EpLibrary> library_in, std::u
   ORT_RETURN_IF_ERROR(instance.library->Load());
   const auto& factories = instance.library->GetFactories();
 
+  // OrtHardwareDevice instances to pass to GetSupportedDevices. sorted by type to be slightly more structured.
+  // the set of hardware devices is static so this can also be static.
+  const static auto sorted_devices = SortDevicesByType();
```

Comment:
```suggestion
  const static std::vector<const OrtHardwareDevice*> sorted_devices = SortDevicesByType();
```
nit: Make it easier for future reader to determine which kind of "device" type this is (since there are so many) :)